docs/sources/explore/trace-integration.md

```diff
@@ -20,9 +20,9 @@ Supported data sources are:
 
 For information on how to configure queries for the data sources listed above, refer to the documentation for specific data source.
 
-## Trace View explanation
+## Trace View
 
-This section explains the elements of the Trace View dashboard.
```

Comment:
Trace View is a quite complex visualization, but calling it a dashboard sounds confusing.